packages/react-scripts/template/src/serviceWorker.js

```diff
@@ -18,7 +18,7 @@ const isLocalhost = Boolean(
     )
 );
 
-export default function register() {
+export function register() {
```

Comment:
The default export still needs to be registerServiceWorker though (even if both are named exports too) for it to be backwards compatible (even if this lands to a major version update 2.0.0, it would be unneccessary to break everyones build for such a tiny change).

Still partial to passing the boolean in myself: https://github.com/facebookincubator/create-react-app/issues/2554#issuecomment-356218165

Reply:
It won't break anyone's build because this part is copied into every project. Even if we changed completely the API between them, we could've shipped that in any patch release IMO.